icons/nodejs/nodejs-plain.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#83CD29" d="M112.678 30.334L68.535 4.729c-2.781-1.584-6.424-1.584-9.227 0L14.82 30.334C11.951 31.985 10 35.088 10 38.407v51.142c0 3.319 1.992 6.423 4.862 8.083l11.729 6.688c5.627 2.772 7.186 2.772 9.746 2.772 8.334 0 12.662-5.039 12.662-13.828v-50.49C49 42.061 49.445 41 48.744 41h-5.622C42.41 41 41 42.061 41 42.773v50.49c0 3.896-3.616 7.773-10.202 4.48L18.676 90.73c-.422-.23-.676-.693-.676-1.181V38.407c0-.482.463-.966.891-1.213l44.378-25.561a1.508 1.508 0 011.415 0l43.963 25.555c.421.253.354.722.354 1.219v51.142c0 .488.092.963-.323 1.198l-44.133 25.576c-.378.227-.87.227-1.285 0l-11.317-6.749c-.341-.198-.752-.269-1.08-.086-3.145 1.783-3.729 2.02-6.679 3.043-.727.253-1.799.692.408 1.929l14.798 8.754a9.29 9.29 0 004.647 1.246 9.303 9.303 0 004.666-1.246l43.976-25.582c2.871-1.672 4.322-4.764 4.322-8.083V38.407c-.001-3.319-1.452-6.414-4.323-8.073zM77.727 81.445c-11.727 0-14.309-3.235-15.17-9.066-.102-.628-.634-1.379-1.274-1.379h-5.73c-.709 0-1.28.86-1.28 1.566 0 7.466 4.06 16.512 23.454 16.512 14.038 0 22.088-5.455 22.088-15.109 0-9.572-6.467-12.084-20.082-13.886-13.762-1.819-15.16-2.738-15.16-5.962 0-2.658 1.184-6.203 11.374-6.203 9.104 0 12.46 1.954 13.841 8.091.119.577.646.991 1.241.991h5.754c.354 0 .691-.143.939-.396.241-.272.367-.613.336-.979-.893-10.569-7.913-15.494-22.112-15.494-12.632 0-20.166 5.334-20.166 14.275 0 9.698 7.497 12.378 19.622 13.577 14.505 1.422 15.633 3.542 15.633 6.395 0 4.956-3.978 7.067-13.308 7.067z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#5fa04e" d="M63.984.002c-1.092-.028-2.203.195-3.18.82l-49.7 28.817.048.091-.05.03C9.208 30.832 8.145 32.856 8.145 35v57.988c0 1.308.444 2.568 1.19 3.598l-.124.213a6.246 6.246 0 0 0 1.656 1.428l49.643 28.697.412.238h.002c.274.155.558.283.846.393.197.079.395.143.597.2.035.01.07.016.104.024a5.596 5.596 0 0 0 2.2.145l.04.07a8.233 8.233 0 0 0 2.248-.715l49.94-29.054c1.42-.834 2.483-2.263 2.956-3.81l-.216-.374c.057-.346.097-.696.097-1.053V35c0-1.522-.605-2.977-1.586-4.08l.05-.088a3.706 3.706 0 0 0-1.304-1.072L91.867 15.293 66.96.824a6.175 6.175 0 0 0-1.873-.68L65.064.11c-.107-.018-.217-.02-.326-.033-.03-.004-.061-.004-.092-.008-.22-.025-.438-.06-.662-.066z"/></svg>
```

Comment:
Same here as with the line versions. Imo this is a better representation:
![nodejs-plain](https://github.com/devicons/devicon/assets/43886029/c069a7b0-c811-4839-b004-a55b5d1e3f35)

I wasn't able to convert it to a single path, but here's the code for it in case you wanna try yourself:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
   <defs>
      <clipPath id="a" clipPathUnits="userSpaceOnUse">
         <path
            d="M3.143-5h121.712v137.994H3.143ZM63.994-.498c-.39-.01-.785.01-1.178.066a5.695 5.695 0 0 0-2.279.834l-50.105 29.05.062.114C8.67 30.776 7.643 32.834 7.643 35v57.988c0 1.305.41 2.555 1.101 3.614l-.14.244.218.267a6.751 6.751 0 0 0 1.79 1.545l.001.002 49.649 28.697.41.239h.002c.307.175.626.317.951.44.185.072.371.131.56.187.08.022.16.038.239.056.65.163 1.314.255 1.98.201l.038.067.351-.06c.754-.128 1.623-.373 2.393-.76l.013-.008 49.951-29.063c1.542-.904 2.675-2.43 3.184-4.096l.065-.209-.237-.406c.045-.316.073-.636.073-.959V35c0-1.53-.557-2.988-1.48-4.143l.038-.072-.197-.256a4.198 4.198 0 0 0-1.477-1.217l.028.014-25.932-14.99L67.207.393V.39a6.549 6.549 0 0 0-1.8-.694l-.026-.043-.233-.039c-.106-.018-.22-.017-.328-.031-.045-.006-.09-.009-.136-.014-.23-.026-.456-.062-.69-.068zm-.058 1.02a5.57 5.57 0 0 0-2.764.738L11.824 29.918l17.26 31.668L64.524.574a8.108 8.108 0 0 0-.583-.053h-.005zm2.298.601 34.899 60.07 16.406-30.195a5.437 5.437 0 0 0-1.021-.81L91.088 15.42 66.615 1.27l-.006-.001c-.12-.061-.25-.097-.375-.147zm-1.015.244L29.645 62.613l35.28 64.729c.099-.02.198-.023.296-.049l35.353-65.072ZM10.973 30.445c-1.49 1.023-2.33 2.728-2.33 4.555v57.988c0 .923.25 1.823.677 2.623l19.176-33.013Zm107.252 1.385-16.502 30.375 17.511 30.143V35c0-1.141-.379-2.247-1.01-3.17zm-17.06 31.404L66.65 126.758c.018-.01.037-.017.055-.027l49.822-28.938.004-.002c1.39-.786 2.281-2.137 2.582-3.662zm-72.108.391L9.939 96.541c.403.496.878.929 1.418 1.256.002 0 .001.003.002.004l49.285 28.623.526.304.006.002h.002c.255.15.523.263.79.368.143.053.286.102.43.144.124.037.249.061.373.088.24.051.479.088.721.108.117.009.235.01.352.011z"
            class="powerclip" />
      </clipPath>
   </defs>
   <path fill="#5fa04e"
      d="M63.984.002c-1.093-.028-2.202.195-3.177.82L11.104 29.64l.05.091-.05.03C9.209 30.832 8.143 32.855 8.143 35v57.988c0 1.308.445 2.568 1.191 3.598l-.123.213a6.251 6.251 0 0 0 1.656 1.43l49.643 28.695.412.238h.002a6.07 6.07 0 0 0 .008.004 5.818 5.818 0 0 0 3.742.758l.037.07c.71-.12 1.54-.358 2.25-.715l49.937-29.054c1.42-.833 2.484-2.263 2.957-3.81l-.218-.376c.057-.346.097-.696.097-1.053V35c0-1.522-.603-2.975-1.584-4.078l.05-.09a3.702 3.702 0 0 0-1.304-1.072L91.451 15.05 66.957.825A6.07 6.07 0 0 0 65.092.15l-.026-.043a7.752 7.752 0 0 0-1.082-.105z"
      clip-path="url(#a)" transform="matrix(1.00004 0 0 1.00004 -.002 0)" />
</svg>
```

Reply:
Great work @lunatic-fox! I wonder how you created the clipping without clipPath though, do you have any tips, and perhaps a guide? :)